File: src/main/java/com/easy/zssn/model/Location.java
package com.easy.zssn.model;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Location {
    @Id
    Integer survivorID;
    double latitude;
    double longitude;

    public double getLatitude() {
        return latitude;
    }
    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }
    public double getLongitude() {
        return longitude;
    }
    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }
    public Integer getSurvivorID() {
        return survivorID;
    }
    public void setSurvivorID(Integer survivorID) {
        this.survivorID = survivorID;
    }
   
}

File: src/main/kotlin/com/eduardo/rdguez/zssn/domain/Location.kt
package com.eduardo.rdguez.zssn.domain

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "location")
class Location(
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  var id: Long? = null,

  @Column(name = "latitude")
  var latitude: Int,

  @Column(name = "longitude")
  var longitude: Int,
)